item_tracker.py

```diff
@@ -29,7 +32,7 @@ def __init__(self, id, x, y, index, shown, floor):
 
 
 # TODO: keep track of all curses
-class Floor:
+class aFloor:
```

Comment:
It was a temporary change to keep it from conflicting with the Floor class in floor.py - I didn't want to remove the old floor class yet because I was still testing the new stuff. I forgot to clean that up when I removed everything from item_tracker.py.